test/utils/ipfs-exec.js

```diff
@@ -34,11 +35,7 @@ module.exports = (repoPath, opts) => {
   }))
 
   const execute = (exec, args) => {
-    if (args.length === 1) {
-      args = args[0].split(' ')
-    }
-
-    const cp = exec(args)
+    const cp = exec(yargs('-- ' + args[0]).argv._)
```

Comment:
oh, what is this doing?

Reply:
This allows us to parse commands like
`ipfs refs --format="<src> -> <dst>"`
(previously we would just split on a space character)

Reply:
👍 could you add a comment to explain please?